packages/server/src/auth/clientinfo.ts

```diff
@@ -0,0 +1,24 @@
+import { badRequest } from '@medplum/core';
+import { ClientApplication } from '@medplum/fhirtypes';
+import { Request, RequestHandler, Response } from 'express';
+import { asyncWrap } from '../async';
+import { sendOutcome } from '../fhir/outcomes';
+import { getSystemRepo } from '../fhir/repo';
+import { rewriteAttachments, RewriteMode } from '../fhir/rewrite';
+
+export const clientInfoHandler: RequestHandler = asyncWrap(async (req: Request, res: Response) => {
+  const { clientId } = req.params;
+  const systemRepo = getSystemRepo();
+
+  try {
```

Comment:
You can safely skip this `try`/`catch`.  We have built-in express middleware that will automatically catch the error, and return the correct 404 error response.